website/docs/docs/cloud/manage-access/invite-users.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Invite users to dbt Cloud"
+description: "Learn how to manually invite users to dbt Cloud"
+id: "invite-users"
+sidebar: "Invite users"
+---
+
+dbt Cloud makes it easy to invite new users to your environment out of the box. This feature is available to all dbt Cloud customers on Teams or Enteprise plans (Developer plans are limited to a single user).
+
+## Invite new users
+
+You must have proper permissions to invite new users:
+
+- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
+- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.
+
```

Comment:
they ned to have a developer license seat too, right? i think we should add that here because I've had users reach out and are read only seatsand owner permissions reach out wondering why they can't invite users (or manage acct settings)